test/module/s3.spec.js

```diff
@@ -124,6 +124,33 @@ describe('Testing s3 Util', { useNock: true, timestamp: 1569876020 }, () => {
     ]);
   });
 
+  it('Testing "listObjects" with "StopAfter"', async () => {
+    const result = await aws.s3.listObjects({
+      bucket,
+      stopAfter: '2020-03-30T15:10:00.000Z'
```

Comment:
This test should include a key that is the exact timestamp of `stopAfter` and it should return